src/main/Catalog.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+
+function Catalog() {
+  let data;
+  const [state, setstate] = useState([]);
+  let xhttp = new XMLHttpRequest();
+  xhttp.onload = function () {
+    if (this.readyState === 4 && this.status === 200) {
+      data = JSON.parse(xhttp.responseText);
+    }
+    setstate(data);
+  };
+  xhttp.open("GET", "https://api.punkapi.com/v2/beers", true);
```

Comment:
If we're planning to use this endpoint further on, I would advise to make it a global `const` and re-use everywhere we need.